filemanager.c

```diff
@@ -0,0 +1,145 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "helpers.h"
+#include "books.h"
+
+
+typedef struct Record {
+    char* data;
+    struct Record* next;
+}Record;
+
+Record* record_head = NULL;
```

Comment:
I think we should create separate record instances for books, transactions and book copies instead of using single record